<?xml version="1.0" ?>
<sdf version="1.4">
  <model name="c_clamp">
    <pose>0 0 0.16  0 0 0</pose>
    <static>false</static>
    <link name="body">
      <inertial>
        <pose>-0.053216 -0.005553 -0.037320  0 0 0</pose>
        <mass>1</mass>
        <inertia>
          <ixx>4.93835e-3</ixx>
          <ixy>-0.01722e-3</ixy>
          <ixz>-1.89640e-3</ixz>
          <iyy>19.23337e-3</iyy>
          <iyz>-0.027796e-3</iyz>
          <izz>14.42026e-3</izz>
        </inertia>
      </inertial>
      <visual name="visual">
        <cast_shadows>true</cast_shadows>
        <geometry>
         <mesh>
           <uri>model://c_clamp/meshes/c-clamp.dae</uri>
         </mesh>
        </geometry>
      </visual>
      <collision name="collision">
        <surface>
          <friction>
            <ode>
              <mu>0.3</mu>
              <mu2>0.3</mu2>
            </ode>
          </friction>
        </surface>
        <geometry>
         <mesh>
           <uri>model://c_clamp/meshes/c-clamp.stl</uri>
         </mesh>
        </geometry>
      </collision>
    </link>
  </model>
</sdf>
